paymail_service_provider.go

```diff
@@ -187,6 +186,25 @@ func (p *PaymailDefaultServiceProvider) RecordTransaction(ctx context.Context,
 	}, nil
 }
 
+func buildBtTx(p2pTx *paymail.P2PTransaction) *bt.Tx {
+	if p2pTx.DecodedBeef == nil {
+		res := p2pTx.DecodedBeef.GetLatestTx()
```

Comment:
When `p2pTx.DecodedBeef` is nil (as you checked in previous line) - will `GetLatestTx` method work on nil object?